src/thor/route_action.cc

```diff
@@ -245,6 +245,11 @@ thor::PathAlgorithm* thor_worker_t::get_path_algorithm(const std::string& routet
 
   // Have to use multimodal for transit based routing
   if (routetype == "multimodal" || routetype == "transit") {
+    const auto& max_walking_dist = options.costings()
+                                       .find(Costing::pedestrian)
```

Comment:
that was a bit confusing, as the docs said it'd belong to the transit options. but really it's parsed in the pedestrian costing, so I changed the docs; IMO it makes more sense on pedestrian.